catroid/src/org/catrobat/catroid/ui/WebViewActivity.java

```diff
@@ -103,6 +107,19 @@ public void onDownloadStart(String url, String userAgent, String contentDisposit
 		});
 	}
 
+	BroadcastReceiver onDownloadComplete =new BroadcastReceiver() {
```

Comment:
Although this has just been merged, a few comments. Please run the formatter on this code, you're missing a space here.